static/css/tools/choropleth.scss

```diff
@@ -18,27 +18,15 @@
 @import '../tools/statvar_menu';
 @import '../draw';
 
-#map_container {
-  margin-left: 5%;
-}
-
 .border {
   stroke: gray;
   stroke-width: 0.5px;
 }
 
-.border-highlighted {
-  stroke: black;
-  stroke-width: 1px;
-}
-
-.clickable {
-  cursor: pointer;
-}
-
 #explore {
   float: left;
   width: 25%;
+  z-index: 5;
```

Comment:
how does the z-index help here?

Reply:
when the window gets too narrow, this prevents the choropleth map from showing up on top of the stat var menu